### PR TITLE
Move some misplaced tests under full package, add CDI_FULL category. …

### LINKED ISSUE #334: Verify that Lite tests are not using a beans.xml content other than bean-discovery-mode attribute to affect behavior
It was brought up on today's spec call that there are Lite tests that are dictating behavior by beans.xml content other than the bean-discovery-mode attribute. These need to be investigated.

### PR DESCRIPTION
**Pull request by manovotn**
…Rework few other tests to work equally for Lite and Full.

Superseeds https://github.com/eclipse-ee4j/cdi-tck/pull/337
Fixes #334

### CODE REVIEW

**Comment by manovotn** [comment]
Also, I should add that I ran the change against embedded Weld:

```
[INFO] Results:
[INFO] 
[INFO] Tests run: 1249, Failures: 0, Errors: 0, Skipped: 0
```

File: impl/src/main/resources/org/jboss/cdi/tck/tests/full/alternative/broken/incorrect/name/beans.xml [hunk 1]
```diff
@@ -4,6 +4,6 @@
          https://jakarta.ee/xml/ns/jakartaee/beans_3_0.xsd"
        version="3.0" bean-discovery-mode="all">
     <alternatives>
-        <class>org.jboss.cdi.tck.tests.alternative.broken.incorrect.name.NonExistingClass</class>
+        <class>org.jboss.cdi.tck.tests.full.alternative.broken.incorrect.name.NonExistingClass</class>
```
Comment (Ladicek) [inline impl/src/main/resources/org/jboss/cdi/tck/tests/full/alternative/broken/incorrect/name/beans.xml:7]: Shouldn't the other `beans.xml` files that have been moved also have a change like this?

Comment (manovotn) [reply impl/src/main/resources/org/jboss/cdi/tck/tests/full/alternative/broken/incorrect/name/beans.xml:7]: @Ladicek not sure I get you, there are 5 beans.xml files moved like this.
The other beans.xml files were either completely empty (for instance it has `<interceptors>` but not listing any) or were referencing non-existent classes (as intended part of the test). Those classes might have had different packages altogether so IDEA didn't change them when I issued a package change :)

Comment (Ladicek) [reply impl/src/main/resources/org/jboss/cdi/tck/tests/full/alternative/broken/incorrect/name/beans.xml:7]: Fair enough, I was just curious :-)

**Review by Ladicek — APPROVED** [review]

**Review by starksm64 — APPROVED** [review]

**Pull request merged**